ffteeg.py

```
@@ -95,68 +95,58 @@ def __init__(self):
-            windowed_data = filtered_window * hamming(len(filtered_window))
-            fft_data = np.abs(fft(windowed_data))[:len(windowed_data) // 2]
-            window_freqs = np.fft.fftfreq(len(windowed_data), 1 / self.sampling_rate)[:len(windowed_data) // 2]
-
-            brainwave_power = self.calculate_brainwave_power(fft_data, window_freqs)
-            self.brainwave_bars.setOpts(height=brainwave_power)
+                raw_point = sample[0]
+                self.filter_buffer.append(raw_point)
+
+                # Apply the filters if the buffer is full
+                if len(self.filter_buffer) == self.filter_buffer_size:
+                    notch_filtered = filtfilt(self.b_notch, self.a_notch, list(self.filter_buffer))[-1]
```

Ruff (0.8.2)

Local variable `notch_filtered` is assigned to but never used

Remove assignment to unused variable `notch_filtered`

(F841)